bin/cgp.sh

```diff
@@ -16,19 +17,26 @@ NETWORK=""
 UPGRADE=""
 PHASE=""
 SIMULATE=false
-while getopts n:u:p:s flag
+USE_FORK=false
+while getopts n:u:p:sf flag
 do
     case "${flag}" in
         n) NETWORK=${OPTARG};;
         u) UPGRADE=${OPTARG};;
         p) PHASE=${OPTARG};;
         s) SIMULATE=true;;
+        f) USE_FORK=true;;
     esac
 done
 
 parse_network "$NETWORK"
 parse_upgrade "$UPGRADE"
 
+if [ "$USE_FORK" = true ] ; then
+    RPC_URL="http://127.0.0.1:8545"
```

Comment:
could be helpful to add a line to the comments above that this requires a anvil node at localhost